app/marketing/management/commands/assemble_leaderboards.py

```diff
@@ -112,11 +119,19 @@ def sum_bounties(b, usernames):
                 breakdown = 'yearly'
                 sum_bounty_helper(b, breakdown, username, val_usd)
 
+        add_element('all_all', username, b._val_usd_db)
```

Comment:
@owocki Looks like we were missing cases when `idx_status` doesn't equal `done` as we were previously.

Reply:
thanks